velox/vector/fuzzer/tests/VectorFuzzerTest.cpp

```diff
@@ -405,7 +405,9 @@ TEST_F(VectorFuzzerTest, row) {
 
   // Composable API.
   vector = fuzzer.fuzzRow(
-      {fuzzer.fuzzFlat(REAL(), 100), fuzzer.fuzzFlat(BIGINT(), 100)}, 100);
+      {fuzzer.fuzzFlat(REAL(), 100), fuzzer.fuzzFlat(BIGINT(), 100)},
+      std::vector<std::string>{"", ""},
```

Comment:
we should try to pass actual names and assert that the returned vector has them properly set.